src/app/vendorEngramsXyzApi/vendorEngramsXyzService.ts

```diff
@@ -0,0 +1,90 @@
+import { VendorDrop, VendorDropType, VendorDropXyz, toVendorDrop } from './vendorDrops';
+import { t } from 'app/i18next-t';
+
+export class VendorEngramsXyzService {
+  refreshInterval: number = 1000 * 60 * 15;
+  cachedResponse: VendorDrop[];
```

Comment:
It'd be great to move all this stuff to Redux and drive it off of actions instead. Then, we could use that state anywhere in the app to render the status of drops.